lib/faker/default/address.rb

```diff
@@ -333,6 +333,27 @@ def longitude
       def full_address
         parse('address.full_address')
       end
+
+      ##
+      # Produces Address hash of required fields
+      #
+      # @return [Hash]
+      #
+      # @example
+      #   Faker::Address.full_address_as_hash(:longitude,
+      #                                       :latitude,
+      #                                       :country_name_to_code,
+      #                                       country_name_to_code: {name: 'united_states'})
+      #     #=> {:longitude=>-101.74428917174603, :latitude=>-37.40056749089944, :country_name_to_code=>"US"}
+      #
+      # @faker.version 2.11.0
```

Comment:
```suggestion
      # @faker.version next
```
New generators should be marked as available in the next version of Faker